base/applications/cmdutils/attrib/attrib.c

```diff
@@ -274,14 +239,15 @@ ChangeAttribute(
             continue;
         }
 
+// FIXME: This was done for ChangeAttribute(), but not for PrintAttribute() ??
+#if 0
         if (bRecurse && bIsDir && !bDirectories)
             continue;
+#endif
```

Comment:
@Doug-Lyons : Do you happen to know why (from your committed PR), the `ChangeAttribute()` function had this extra piece of code, while the `PrintAttribute()` function did not?

Reply:
@HBelusca, It has been some time since I have worked on this code, so I am not sure that my memory is correct on this. But I believe that this was related to Thomas Fabers comment: "Directories seem to be excluded whenever wildcards are used, so attributes are only applied to them if mentioned specifically by name"(https://github.com/reactos/reactos/pull/2142). I think that the regression tests will reflect the correct usage, so running your changes against the cmd:attrib test should tell you whether your changes are correct or not.

Reply:
@HBelusca, I added your "if 0/#endif" around these two lines in my source for attrib.c and ran the regression test for cmd:attrib. These new results show 12 failures with this code removed as you can see here: 

![new_attrib_fails_01](https://user-images.githubusercontent.com/32620577/147837998-a21c060e-94df-427d-8e21-592e59cd157f.png)

Reply:
> I added your "if 0/#endif" around these two lines in my source for attrib.c 

In the one currently in master?

And out of curiosity, what would happen instead if you were to **add** this extra code (that is #if-0'ed in this PR), not only in `ChangeAttribute()` as it currently is, but in `PrintAttribute()` as well?

Reply:
@HBelusca, Yes, I tested this on the 'attrib.c' file that is currently in master. I changed the code as you asked and added this additional code to `PrintAttributes()` and it seems not to cause any failures. So it looks safe to add it. Here is the result of running with the added code:

![new_attrib_OK_01](https://user-images.githubusercontent.com/32620577/147838496-77331ad5-43fc-4a7b-ac49-83b78f3af054.png)

Reply:
Thanks for the tests!